pyro/distributions/torch.py

```diff
@@ -324,6 +333,7 @@ def expand(self, batch_shape):
 
     __all__.append(_name)
 
+__all__.append('TransformModule')
```

Comment:
I think this can be removed since you are adding this in `distributions.__init__.py`. 

nit: I think a better place for this class might be `torch_distribution.py` which has other mixin classes too.